util/check-format-test-negatives.c

```diff
@@ -22,9 +23,19 @@
         return;                                 \
     }
 
+/* allow extra  SPC in single-line comment */
 /*-
- * allow extra SPC in format-tagged multi-line comment
+ * allow extra  SPC in regular multi-line comment
  */
+/*-
+ * allow extra  SPC in format-tagged multi-line comment
+ */
+/** allow extra '*' in comment opening */
+/*! allow extra '!' in comment opening */
+/*
+ ** allow "**" as first non-space chars of a line within multi-line comment
+ */
```

Comment:
These comment openings are new and note quite our coding style.  Have you found them in aged code?

Reply:
There are various occurrences of `**` in the beginning of comment lines - see the output of 
```
find . -name '*.[ch]'  | xargs grep "^[/ ]\*\*[/ ]"
```
while the `/!*` are found only in `krb5/` - see the output of 
```
find . -name '*.[ch]'  | xargs grep "^/\*\!"
```
but they are usual also in many other projects like https://github.com/siemens/libsecutils that make use of Doxygen.

If you prefer, I can make the acceptance of these styles optional (off by default).
Or may it make sense to explicitly allow (part or all of) these in the coding style guidelines?
At least, they are not explicitly forbidden in its _Chapter 8: Commenting_.

Reply:
I get it that they do exist a little here and there.  As for options, I personally prefer to be flexible by default unless our coding style really forbids something, *but* you could choose to have a "strict" option that's harsher.  Your call.

Reply:
Pleased to hear. 
Then my preference is, for simplicity:
* not to add a "strict comment" option
* not to touch the coding style, which neither encourages nor forbids these special comment styles.